packages/dart_pptx/lib/src/classes/slide.g.dart

```diff
@@ -20,6 +20,4 @@ Map<String, dynamic> _$SlideToJson(Slide instance) => <String, dynamic>{
       'notesId': instance.notesId,
       'background': instance.background.toJson(),
       'hasNotes': instance.hasNotes,
-      'imageRefs':
```

Comment:
All the changes related to the files with `.g.dart` extensions are auto-generated with the `dart run build_runner build` command.